README.md

```diff
@@ -253,6 +257,21 @@ For remote or external Docker Engines—or if you prefer not to mount the Docker
 
 That's it!
 
+## Nextcloud 32: Migrating Existing ExApps from DSP to HaRP
+
+> **Note:** All ExApps developed by Nextcloud will support HaRP when Nextcloud 32 is released. We hope that most ExApps from the community will also support it. **Contact us if you need assistance.**
+
+If you've upgraded to Nextcloud 32 and want to switch from using DSP to HaRP, follow these steps:
+
+1. **Install HaRP** on the same Docker Engine that you were using for DSP.
+2. **Test Deployment on HaRP** with usual `TestDeploy` button.
+3. **Set HaRP as the default deployment daemon** for ExApps.
+4. **Remove the ExApps without deleting their data volumes**:
+   - **Terminal**: Do not use the `--rm-data` option when removing the app.
+   - **From UI**: Do not use the "Delete data when removing" checkbox.
+5. **Install the ExApp**: Install removed ExApps, now they will be installed on `HaRP`.
+6. **Remove DSP**: Now you don't need DSP, and it's safe to remove it.
```

Comment:
let's clarify a bit more that the previous DSP (Docker Socket Proxy) can be removed safely

Reply:
done